src/lambda/io.rs

```diff
@@ -27,6 +27,9 @@ pub enum IO {
         req_template: http::RequestTemplate,
         group_by: Option<GroupBy>,
         dl_id: Option<DataLoaderId>,
+        /// Provides us the ability to pass request
+        /// interception handler via on_request field.
+        http_filter: Option<http::HttpFilter>,
```

Comment:
should not be optional. Keep a default value if it's not available.

Reply:
ok